elastica/wrappers.py

```diff
@@ -1,19 +1,5 @@
 import warnings
 
-__all__ = [
-    "BaseSystemCollection",
-    "Connections",
-    "Constraints",
-    "Forcing",
-    "CallBacks",
-    "Damping",
-]
-from elastica.modules.base_system import BaseSystemCollection
-from elastica.modules.connections import Connections
-from elastica.modules.constraints import Constraints
-from elastica.modules.forcing import Forcing
-from elastica.modules.callbacks import CallBacks
-from elastica.modules.damping import Damping
```

Comment:
we may need to keep this for backward compatibility. @AsadNizami what was the reasoning to remove this?

Reply:
> Please check the formatting failing tests and fix them.

![image](https://user-images.githubusercontent.com/60483464/230293486-f373d5cb-2466-4d37-8dde-732b978c4c55.png)


The unused import statements were causing the formatting test to fail.

Reply:
We can mention the imports in the `__all__` keyword to fix the issue.

Reply:
I see thanks @AsadNizami for the clarification. Let's bring back the `__all__` then for backward compatibility. There may still be some users that refer to the `modules` as `wrappers`.

Reply:
Sure thing!